staging/src/k8s.io/kubectl/pkg/cmd/version/version.go

```diff
@@ -162,11 +164,14 @@ func (o *Options) Run() error {
 	}
 
 	if versionInfo.ServerVersion != nil {
-		if err := printVersionSkewWarning(o.ErrOut, *versionInfo.ClientVersion, *versionInfo.ServerVersion); err != nil {
+		warningMessage, err := printVersionSkewWarning(o.ErrOut, *versionInfo.ClientVersion, *versionInfo.ServerVersion)
+		if err != nil {
 			return err
 		}
+		if o.CheckVersionSkew && warningMessage != "" {
+			return fmt.Errorf("CheckVersionSkew: %q", warningMessage)
```

Comment:
I believe the Go standard is for errors to start with lowercase.  I'm not sure we need to indicate anything here,  Would this work?

```suggestion
			return errors.New(warningMessage)
```

Or do you think we need more information in the message?

Reply:
You are right, go standard is start with lower case for errors. Adjusted in e70aade49a316bb1df13cfda6a86eddcd9ed7364